userspace/libsinsp/threadinfo.cpp

```diff
@@ -1362,7 +1411,7 @@ void sinsp_thread_manager::increment_mainthread_childcount(sinsp_threadinfo* thr
 		//
 		ASSERT(threadinfo->m_pid != threadinfo->m_tid);
 
-		sinsp_threadinfo* main_thread = m_inspector->get_thread_ref(threadinfo->m_pid, create_if_needed, true).get();
+		sinsp_threadinfo* main_thread = m_inspector->get_thread_ref(threadinfo->m_pid, true, true).get();
```

Comment:
I'm pretty sure we want to keep this as `create_if_needed`, this is one of our changes.